ProgramFlow.info

```diff
@@ -1,17 +1,52 @@
-This file is meant to document the flow of information in the codebase.
+This file is meant to help a developer navagate the codebase and develop clean,
+maintainable code.
 
-Within the consensus package, it is convention to use the term 'apply' any time
-that a function is changing the state as the result of information being
-introduced to the blockchain. For example 'ApplyBlock' and 'ApplyStorageProof'.
+///////////////////////////
+// Consensus Conventions //
+///////////////////////////
 
-Within the consensus package, it is convention to use the term 'invert' any
-time that a function is changing the state as the result of information being
-removed from the blockchain. For example 'invertBlock'.
+It is convention to use the term 'apply' any time that a function is changing
+the state as the result of information being introduced to the state. For
+example 'ApplyBlock' and 'applyStorageProof'.
 
-Within the consensus package, it is convention to use the term 'valid' any time
-that the validity of something is being checked. It is convention not to change
-the state in any capacity when inside of a validation function. For example
-'validInput' and 'validContract'.
+It is convention to use the term 'invert' any time that a function is changing
+the state as the result of information being removed from the blockchain. For
+example 'invertBlock'.
```

Comment:
invert was actually confusing me because it looks a lot like insert.

I'll probably switch to revert when I rewrite the consensus package, which will be further down the road.